VCFv4.3.tex

```diff
@@ -564,23 +583,28 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 \end{samepage}
-This key can be derived from the REF/ALT fields but is useful for filtering.
-The reserved values must be used for the types listed below:
+SVTYPE is a primitive basic data type and must be specified for all structural variants.
+Its value can usually be deduced from the REF and ALT fields, however despite this redundancy it is quite useful for filtering.
+For each VCF record, SVTYPE must be assigned exactly one of the following values:
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
-  \item BND: Breakend
+\item DEL: Deletion relative to the reference
+\item INS: Insertion relative to the reference
+\item DUP: Region of elevated copy number relative to the reference
```

Comment:
Is it intentional that the description for DUP is different here in SVTYPE than the one in symbolic ALTs? I would change it here too.

Reply:
Right now this seems to indicate that if you know there's a duplication but aren't sure if it's tandem or not, you should use `SVTYPE=DUP` and have an `ALT` of `<CNV>`. Is that the intent?

Reply:
Also, there are VCFs out there that use `<DUP>` to indicate regions of elevated copy number rather than tandem duplications. Perhaps we should call out a warning about interpreting legacy VCFs given this change.